Recrutify/Recrutify.Services/Helpers/StaffHelper.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Recrutify.DataAccess;
+using Recrutify.DataAccess.Models;
+
+namespace Recrutify.Services.Helpers
+{
+    public class StaffHelper
```

Comment:
make interface, registrate in AddHelpers extension and methods shouldn't be static

Reply:
+